docs/sources/setup-grafana/configure-grafana/enterprise-configuration/index.md

```diff
@@ -174,6 +174,10 @@ Name of the TrueType font file with italic style.
 
 Maximum number of panel rendering request retries before returning an error. To disable the retry feature, enter `0`. This is available in public preview and requires the `reportingRetries` feature toggle.
 
+### allowed_domains
+
+Allowed domains to receive reports. Use * to allow all domains. Use a comma-separated list to allow multiple domains. Example: allowed_domains = grafana.com, example.org
```

Comment:
I realize I didn't make this change on the corresponding enterprise PR. Sorry! We should eventually make the same change on that side.
```suggestion
Allowed domains to receive reports. Use an asterisk (`*`) to allow all domains. Use a comma-separated list to allow multiple domains. Example: allowed_domains = grafana.com, example.org
```